sdk/include/opentelemetry/sdk/metrics/state/observable_registry.h

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #pragma once
+#include <future>
```

Comment:
Nit - Can we move this header within enable macro if it is required.

Reply:
thanks, cleaned